src/app/features/landing/components/landing-hero/landing-hero.component.html

```diff
@@ -0,0 +1,25 @@
+<section id="hero" class="hero-container">
+  <div class="grid">
+    <div class="col-md-6">
+      <header class="hero-content">
+        <h1 class="hero-content-header">
+          <span class="font-light block">Lorem ipsum dolor</span>consectetur adipiscing elit
```

Comment:
Lets these text values from the i18n language json file instead with translation service.
And keep them descriptive for now. Like her header text, hero header description.

Reply:
I have added localization feature to the app in my latest commit. The localization has following features

1. For the first time visitor detects browser language and translate the app to that language if available.
2. User get an option to choose the languages in the footer.
3. The app gets translated to chosen language and the preference gets saved until the user changes it.